share/csm_share/shr/shr_flux_mod.F90

```diff
@@ -267,7 +277,15 @@ real(R8)    :: psix2  ! stability function at ztref reference height
      if (mask(n) /= 0) then
     
         !--- compute some needed quantities ---
-        vmag   = max(umin, sqrt( (ubot(n)-us(n))**2 + (vbot(n)-vs(n))**2) )
+
```

Comment:
This is getting a little long for the PR review so I'll move it to a separate issue. #1105